pom.xml

```diff
@@ -90,6 +91,8 @@
         <tomcatDistributionUrl>https://archive.apache.org/dist/tomcat/tomcat-8/v8.0.48/bin/apache-tomcat-8.0.48.tar.gz</tomcatDistributionUrl>
 
         <forge.folderctxmenus.version>3.0.1</forge.folderctxmenus.version>
+        <!-- TO BE REMOVED WHEN UPGRADING FROM 12.3.0 -->
+        <hippo.jackrabbit.version>2.16.1-h2</hippo.jackrabbit.version>
```

Comment:
In other words only applicable/used in 12.3.0?  If we then move to 12.4.0 or 13.0.0 for example, the line needs to be removed?  How about upgrade to 12.3.1?

Reply:
Correct, this issue is fixed in 12.3.1 and above as reported by @giacomolm so any upgrade should remove it

Reply:
cool thanks. Also noticed, this has 2 commits?

Reply:
Was half @giacomolm and half me so I left the two to show who did what 🙂

Reply:
👍